modules/social_features/social_group/modules/social_group_flexible_group/modules/social_flexible_group_book/social_flexible_group_book.install

```diff
@@ -43,31 +43,32 @@ function social_flexible_group_book_install(): void {
   $group_permissions['flexible_group-outsider'] = $group_permissions['flexible_group-anonymous'];
 
   // Group member.
-  $group_permissions['flexible_group-member'] = $group_permissions['flexible_group-outsider'];
-
-  // Group manager.
-  $group_permissions['flexible_group-group_manager'] = [
-    ...$group_permissions['flexible_group-member'],
+  $group_permissions['flexible_group-member'] = [
+    ...$group_permissions['flexible_group-outsider'],
     'create group_node:book content',
     'create group_node:book entity',
     'delete own group_node:book entity',
     'delete own group_node:book content',
     'update own group_node:book content',
     'update own group_node:book entity',
   ];
-  // Group outside role: Content manager.
-  $group_permissions['flexible_group-83776d798'] = $group_permissions['flexible_group-group_manager'];
 
-  // Group admin.
-  $group_permissions['flexible_group-group_admin'] = [
-    ...$group_permissions['flexible_group-group_manager'],
+  // Group manager.
+  $group_permissions['flexible_group-group_manager'] = [
+    ...$group_permissions['flexible_group-member'],
```

Comment:
This looks like we're escalating permissions from a group admin to a group manager here.

My proposal would be to not move all of the permissions, but instead mirror what we currently do for topics and events.

In general we've given group_manager's the ability to delete content so they can keep content clean: https://github.com/goalgorilla/open_social/blob/main/modules/social_features/social_group/modules/social_group_flexible_group/config/install/group.role.flexible_group-group_manager.yml

We've limited the ability to edit content to group_admin's since it would also allow for impersonating a user: https://github.com/goalgorilla/open_social/blob/main/modules/social_features/social_group/modules/social_group_flexible_group/config/install/group.role.flexible_group-group_admin.yml

I'm also missing an update hook for the permission changes in this block.

Reply:
Rebuild permissions structure in [the commit](https://github.com/goalgorilla/open_social/pull/3053/commits/e0b274c2104623c8d1bb0973f05e66cde5a2dfb0)